### ISSUE TITLE
[Fix] Change the theme of Simon-Game

### ISSUE THREAD

**Issue by Nitesh-thapliyal** [Bauddhik-Geeks/Simon-Game#1]
- Change background 
- Change the theme of the boxes

**Comment by rhvsingh** [comment]
I want to work on this issue.

**Comment by siddhiiguptaa** [comment]
Hi Nitesh, I would like to work on this issue!

**Comment by Nitesh-thapliyal** [comment]
> Hi Nitesh, I would like to work on this issue!

cool go ahead👍

**Comment by siddhiiguptaa** [comment]
Hi @Nitesh-thapliyal I was looking at this issue, and needed to confirm what exactly you meant by theme? Is it the BG color or some other modifications you want with the boxes?

**Comment by Nitesh-thapliyal** [comment]
> Hi @Nitesh-thapliyal I was looking at this issue, and needed to confirm what exactly you meant by theme? Is it the BG color or some other modifications you want with the boxes?

like right now you can check how to looks here https://bauddhikgeeks.tech/Simon-Game , you can change background, add animation something like that.

**Comment by siddhiiguptaa** [comment]
> > Hi @Nitesh-thapliyal I was looking at this issue, and needed to confirm what exactly you meant by theme? Is it the BG color or some other modifications you want with the boxes?
> 
> like right now you can check how to looks here https://bauddhikgeeks.tech/Simon-Game , you can change background, add animation something like that.

Yes I checked, we can definitely change the background of the website, but should we change the theme of the boxes? 🤔 I checked online, Simon game boxes has the current theme only! Just wanted to confirm!

**Comment by amankumar6** [comment]
I would like to work on this issue!
